### PR TITLE
fix(ui): notification badge & browser notifications

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes :

I worked on the fixing issue around notification..
Notification badge doesn't shows up 

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
Screenshot 2022-07-29 at 8 04 12 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/AppStateProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+      // reactLocalStorage.setObject(LOCALSTORAGE_USER_PROFILES, {
+      //   data: this.userProfilePics,
+      // });
+
+      // return profile;
+    },
+    state.userProfilePics
+  );
+
+  //   updateProfilePicsLoading(id?: string, username?: string) {
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/AppStateProvider.tsx:218]: Can we remove the unwanted code?

File: openmetadata-ui/src/main/resources/ui/src/components/containers/PageContainerV1.tsx [hunk 2]
```diff
@@ -24,9 +24,12 @@ const PageContainerV1 = ({
   className = '',
 }: PageContainerV1Props) => {
   return (
-    <div className={classNames('page-container-v1 tw-bg-body-main', className)}>
-      {children}
-    </div>
+    <>
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/containers/PageContainerV1.tsx:27]: do we need `<>` as we are already returning `<div>`

File: openmetadata-ui/src/main/resources/ui/src/AppStateProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,348 @@
+import { isEmpty, isNil } from 'lodash';
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/AppStateProvider.tsx:1]: can you add a licence for new component

File: openmetadata-ui/src/main/resources/ui/src/components/containers/PageContainerV1.tsx [hunk 4]
```diff
@@ -24,9 +24,12 @@ const PageContainerV1 = ({
...
+      <div
+        className={classNames('page-container-v1 tw-bg-body-main', className)}>
+        {children}
+      </div>
+    </>
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/containers/PageContainerV1.tsx:32]: do we need <> as we are already returning <div>

**Comment by chirag-madlani** [comment]
@ShaileshParmar11 @vivekratnavel AppStateProvider not supposed to commit in this one. Thanks for highlighting it 🙏

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**